docs/cli/fp-run-and-shell-to-cell.md

```diff
@@ -4,28 +4,17 @@ category: 63d7e8bf2482560ba8b126e0
 slug: fp-run-and-shell-to-cell
 ---
 
-# fp run and Shell to Cell
+If you’re an SRE or infrastructure engineer - chances are you spend much of your work time in your shell. Besides querying providers from the Fiberplane notebook, you can easily send your shell comand and output data to a notebook to examine and share with your colleagues.
```

Comment:
Do I understand correctly that `readme.io` has no way to disable the insertion of newline tags? That would seem a violation of the Markdown spec to me. But I suppose this will work for now...

Reply:
Yeah unfortunately, it seems to add a newline on every newline character as opposed to on `\n\n`..